core/modules/widgets/navigator.js

```diff
@@ -540,7 +541,7 @@ NavigatorWidget.prototype.handleImportTiddlersEvent = function(event) {
 	newFields.text = JSON.stringify(importData,null,$tw.config.preferences.jsonSpaces);
 	this.wiki.addTiddler(new $tw.Tiddler(importTiddler,newFields));
 	// Update the story and history details
```

Comment:
May be there should be 1 line of comment here that the `paramObject` could be `event` from above. I think than it's OK the event changed the name.

Reply:
Thanks. I added a comment